packages/rich-text/src/test/styleManipulation.ts

```diff
@@ -17,62 +17,81 @@
 /**
  * External dependencies
  */
-import { convertFromRaw, EditorState, SelectionState } from 'draft-js';
+import {
+  convertFromRaw,
+  EditorState,
+  RawDraftContentState,
+  SelectionState,
+} from 'draft-js';
 
 /**
  * Internal dependencies
  */
+import type { OrderedSet } from 'immutable';
```

Comment:
Isn't this external dependencies

Reply:
In tests. Is there any downside to this?